src/main/java/frc/robot/RobotContainer.java

```diff
@@ -241,6 +243,9 @@ private void configureButtonBindings() {
                             new Pose2d(drive.getPose().getTranslation(), new Rotation2d())),
                     drive)
                 .ignoringDisable(true));
+
+    // While holding the left bumper, use right Y for coral wrist
+    controller.leftBumper().whileTrue(CoralToolCommands.moveByJoystick(coralTool, () -> -controller.getRightY()));
```

Comment:
It is possible that you may need to multiply the number returned by `-controller.getRightY()`, because that will be a number in the range of `[-1, 1]` (inclusive) and that number is fed directly into the voltage control. Multiplying it by 12 (or 6, depending on the speed you want availble) should show you the real speed range of the motor.

```suggestion
    controller.leftBumper().whileTrue(CoralToolCommands.moveByJoystick(coralTool, () -> -controller.getRightY() * 12));
```